coralproject/talk
# [CORL-2768] log the sort order when the comment stream is queried

## What does this PR do?

- adds a log record for the sort order (`orderBy`) when loading the comment stream
    - can be identified either by:
        - log message: `"stream sort order"`
        - traceID: `"SORT_ORDER_LOGGING"`
- NOTE: because of the way Coral queries the stream, this will create two log entries for every one time loading/viewing the stream
    - if this is used for metrics, divide your numbers by 2 and you'll get the accurate number of stream sorts + loads for our users

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- load up the stream
- check Coral logs for traceID: `SORT_ORDER_LOGGING`
- ensure that sort order matches the sort order you loaded in stream
- change the sort order in stream
- see that it is also logged in Coral logs
 
## How do we deploy this PR?

No special considerations.
